Resources/Prototypes/Entities/Objects/Misc/rubber_stamp.yml

```diff
@@ -224,7 +224,7 @@
 
 - type: entity
   name: warden's rubber stamp
-  parent: RubberStampBase
+  parent: [RubberStampBase, BaseRestrictedContraband]
```

Comment:
like before, i dont think this makes sense to list as just restricted. contraband has specific support for depts and potentially(?) individual jobs. if something is meant to be restricted to one job, then it should clearly say that.